devicetypes/smartthings/smartsense-temp-humidity-sensor.src/smartsense-temp-humidity-sensor.groovy

```diff
@@ -26,6 +26,7 @@ metadata {
 		capability "Sensor"
 
 		fingerprint endpointId: "01", inClusters: "0001,0003,0020,0402,0B05,FC45", outClusters: "0019,0003"
+		fingerprint profileId: "0104",deviceId: "0302", inClusters: "0000,0001,0003,0402", model: "b467083cfc864f5e826459e5d8ea6079", deviceJoinName: "Orvibo Temperature & Humidity Sensor"
```

Comment:
The fingerprint should include the manufacturer as well.

nit: put a space before `deviceId`